specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/vulnerabilityAssessment.json

```diff
@@ -0,0 +1,397 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Security Center",
+        "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+        "version": "2015-06-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Security/vulnerabilityAssessment": {
+            "get": {
+                "x-ms-examples": {
+                    "Get vulnerability assessment onboarding status on resource": {
+                        "$ref": "./examples/VulnerabilityAssessment/GetVulnerabilityAssessment_example.json"
+                    }
+                },
+                "description": "Gets the vulnerability assessment status on a given resource.",
+                "tags": [
+                    "VulnerabilityAssessment"
+                ],
+                "operationId": "GetStatus",
+                "parameters": [

```

Comment:
this section should only contain references and not direct declarations
these won't be needed when you change to resourceId